buildSrc/src/main/resources/minimumCompilerVersion

```diff
@@ -1 +1 @@
-14
+1.8
```

Comment:
Sadly, this won't fly, just checked it locally before merge):

```
* What went wrong:
A problem occurred evaluating project ':buildSrc'.
> At least Java 11 is required to build opensearch gradle tools
```

It seems like 11 is the minimum one

Reply:
Added 11, thanks!